fluent-plugin-google-cloud.gemspec

```diff
@@ -19,6 +19,7 @@ eos
   gem.test_files    = gem.files.grep(/^(test)/)
   gem.require_paths = ['lib']
 
+  gem.add_runtime_dependency 'chronic_duration', '~> 0.10.6'
```

Comment:
Asked [Steven](https://github.com/sqrrrl) who is a major contributor of [google-api-ruby-client](https://github.com/google/google-api-ruby-client). It's OK to have dependencies on 3rd party libraries. It's a common thing in our open source projects and API clients. Note that the licenses do matter though. Affero General Public License ([AGPL](https://en.wikipedia.org/wiki/Affero_General_Public_License)) is not allowed.